tests/unit/test-class-wc-payments.php

```diff
@@ -107,7 +102,7 @@ public function test_rest_endpoints_validate_nonce_if_platform_checkout_feature_
 		$this->assertEquals( 'woocommerce_rest_missing_nonce', $response->get_data()['code'] );
 	}
 
-	public function test_rest_endpoints_do_not_validate_nonce_if_platform_checkout_feature_flag_is_enabled() {
+	public function test_rest_endpoints_do_validate_nonce_if_platform_checkout_feature_flag_is_enabled() {
 		// Enable dev mode so nonce check is disabled.
 		add_filter(
 			'wcpay_dev_mode',
```

Comment:
Should we remove this? It was added by @reykjalin in #4115 when the "disable nonce check" was introduced.

Reply:
This was a quick fix to keep the transactions working on our dev environments without adding a potential security risk to live environments. This is no longer needed now that the nonce verification works (dev or live mode).
This PR you are reviewing is a continuation of @reykjalin work.
p1649426189773499-slack-C022WMN88KG

Reply:
> This is no longer needed now that the nonce verification works

So, should we remove the code: 

```php
add_filter(
	'wcpay_dev_mode',
	function () {
		return true;
	}
);
```

Reply:
yes, I'll remove it. Thank you for spotting that.